### PR TITLE
feat(view): add view-manager, add pager, update readme

### PR DESCRIPTION
**Pull request by VMBindraban**

### CODE REVIEW

File: src/bootstrap/data-table.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<template>
+  <require from="./columns-filter"></require>
```
Comment (jeremyvergnas) [inline src/bootstrap/data-table.html]: I think you need to change the path here.

`../../columns-filter` I guess?

Comment (VMBindraban) [inline src/bootstrap/data-table.html]: Fixed.

**Comment by jeremyvergnas** [comment]
I wonder, shouldn't we have a `framework` folder for the view-manager?

**Comment by VMBindraban** [comment]
De data-table.html is located in `bootstrap`, that is the default framework.

**Comment by jeremyvergnas** [comment]
looks good 👍

**Comment by VMBindraban** [comment]
it's your branch :P

**Comment by jeremyvergnas** [comment]
👍

**Pull request merged**